src/renderer.ts

```diff
@@ -419,9 +419,40 @@ const render = (format: Format, imageMappings: ImageMappings, excludeStyles = fa
             const { html, css: styles, js } = element;
 
             const atom = h(InteractiveAtom, { html, styles, js, format });
+            const fenced = `
+                <html>
+                    <head>
+                        <meta charset="utf-8">
+                        <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+                        <script>
+                            const fonts = [].slice.apply(window.parent.document.styleSheets)
+                                .filter(sheet => sheet.ownerNode.attributes?.getNamedItem('data-emotion-css')?.value?.includes('PageStyles'))
+                                .map(sheet => sheet.ownerNode.textContent)
+                                .join(' ')
+                            var css = document.createElement('style');
+                            css.textContent = fonts;
+                            document.head.appendChild(css);
```

Comment:
Is the purpose of this is to make the fonts available in the iframe? If so, since we _have_ the list of `@font-face` rules available on the server, would it make sense to insert them here directly rather than relying on client-side JS?

Reply:
Good idea, that makes sense